clients/go/rest/options.go

```diff
@@ -0,0 +1,45 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/gojek/heimdall/v7"
+
+	raccoon "github.com/odpf/raccoon/clients/go"
+	"github.com/odpf/raccoon/clients/go/serializer"
+	"github.com/odpf/raccoon/clients/go/wire"
+)
+
+// RestClient is the http implementation
+type RestClient struct {
+	raccoon.Client
+	Url        string
```

Comment:
@AkbaraliShaikh i think most of these methods can be private.

Reply:
Yes,  will adjust the protection level accordingly for all the options.

Reply:
updated